ffmpeg/videoprofile.go

```diff
@@ -36,6 +36,25 @@ const (
 	GOPInvalid = -2
 )
 
+// for sceneclassification
+const (
+	DnnClassify = iota
+	DnnYolo
```

Comment:
livepeer/FFmpeg#1 only mentions the lvpclassify and lvpodetect filter types. Do we support these additional types as well?